docs/apps/vtune.md

```diff
@@ -1,7 +1,6 @@
 # Intel VTune Profiler
 
-Intel VTune Profiler is provided via the `intel-vtune` module. One sets up the
-environment by loading the module:
+Intel VTune Profiler is available only on Puhti and it is provided via the `intel-vtune` module. At this moment it support only the applications compiled with Intel MPI library. One sets up the environment by loading the module:
```

Comment:
Is the "Intel MPI library" requirement also necessary   if you want to profile something other than the MPI behavior?

Reply:
At least MPICH works also, about hpcx (OpenMPI) I do not know but it should be at least tested.

Reply:
Single process profiling works as expected for a test application compiled with `hpcx-mpi`

Reply:
Hello,

thank you for looking over the documentations. I misunderstood the term "support" on the Intel documentation page.. That sentence can be removed.